web/vtadmin/src/components/routes/tablets/InfoDropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import Dropdown from '../../dropdown/Dropdown';
+import MenuItem from '../../dropdown/MenuItem';
+import { Icon, Icons } from '../../Icon';
+import PingDialog from './PingDialog';
+
+interface InfoDropdownProps {
+    alias: string;
+    clusterID?: string;
+}
+
+const InfoDropdown: React.FC<InfoDropdownProps> = ({ alias, clusterID }) => {
+    const [isPingOpen, setPingOpen] = useState(false);
```

Comment:
I think I see where you're going with this, and some refactoring will be good down the line. If it's cool with you, I'd rather refactor to be more flexible when I implement the next dialog, because that will give me a better sense of what's necessary 😄

Reply:
Yeah for sure!